tests/zfs-tests/include/libtest.shlib

```diff
@@ -3089,25 +3118,31 @@ function zed_setup
 	# Scripts must only be user writable.
 	saved_umask=$(umask)
 	log_must umask 0022
-	log_must cp ${ZEDLET_LIBEXEC_DIR}/all-syslog.sh $ZEDLET_DIR
-	log_must cp ${ZEDLET_LIBEXEC_DIR}/all-debug.sh $ZEDLET_DIR
 	log_must umask $saved_umask
```

Comment:
You need to move the `umask` changes along with the `cp`.  It's important that the copied scripts only be user writable.